app/models/manageiq/providers/inventory.rb

```diff
@@ -106,11 +121,5 @@ def self.parsed_manager_name(target)
     rescue ManageIQ::Providers::Inflector::ObjectNotNamespacedError => _err
       nil
     end
-
-    # Multiple parser classes
-    # Can be implemented in subclass when custom set needed (mainly for TargetCollection)
-    def self.parser_classes_for(ems, target)
```

Comment:
Only if you need to change the code to fix the travis failure...

Is it possible to keep the method back here?
if you moved it for a reason, that is good, but if there is no reason, then it is easier to navigate fewer changes.
thnx

Reply:
no problem